### ISSUE TITLE
Stratux AHRS Support 

### ISSUE THREAD

**Issue by pophop99** [apps4av/avare#324]
Will you be providing support for your PFD feature leveraging stratux?

**Comment by cyoung** [comment]
A webview might work.

![screen shot 2018-07-18 at 9 29 32 am](https://user-images.githubusercontent.com/57193/42884751-267ab946-8a6d-11e8-8aeb-8cbb9a8ebe14.png)

**Comment by JohnW4SBA** [comment]
> Just thought your avare PFD page could leverage the same data

Makes sense to me, but I've no clue about the tech details involved.  I 
don't know of anyone working on it, but having the html available would 
seem to me quite helpful in mushing the StratuX/AvarePFD code together 
somehow.  Hopefully it will happen.

Happy Flying  -j

**Comment by cyoung** [comment]
The HTML for the AHRS display in Stratux? It's available [here](https://github.com/cyoung/stratux/blob/master/web/plates/gps.html).

It is rendered using JavaScript-generated SVG, [code here](https://github.com/cyoung/stratux/blob/master/web/plates/js/ahrs.js).

Let me know if you need anything else.

**Comment by digivation** [comment]
Is the information from https://github.com/cyoung/stratux/blob/master/notes/app-vendor-integration.md `getSituation` useful? Seems like it would be better to drive the PFD displays already in the app, instead of adding another webview, unless I'm misunderstanding something.
